game/templates/game/board.html

```
@@ -289,6 +298,18 @@
         </div>
     </div>
 
+    <div class="promo-overlay" id="fenOverlay">
+        <div class="promo-dialog">
+            <div class="promo-title">Start from FEN</div>
+            <input type="text" id="fenInput" class="fen-input" placeholder="Paste FEN string">
+            <div class="fen-error" id="fenError"></div>
+            <div class="fen-actions">
+                <button class="btn btn-gold" id="fenStartBtn" style="width: 110px;">Start</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)